cli/start.go

```diff
@@ -94,7 +122,7 @@ type prepStartWorkspaceArgs struct {
 func prepStartWorkspace(inv *clibase.Invocation, client *codersdk.Client, args prepStartWorkspaceArgs) ([]codersdk.WorkspaceBuildParameter, error) {
 	ctx := inv.Context()
 
-	templateVersion, err := client.TemplateVersion(ctx, args.Template.ActiveVersionID)
+	templateVersion, err := client.TemplateVersion(ctx, args.TemplateVersionID)
```

Comment:
this is a good catch - previously we'd be computing the parameters against the active version but then starting the build with the previous version